### PR TITLE
This is a first stab to updating the documentation

### PR DESCRIPTION
**Pull request by JIMENOFONSECA**
@daren-thomas  Could you please verify this? I think this moves around parts of your work . Thanks!

This could be a first stab to the user story #1607 

I am missing just the part about how to name variables.

@BhargavaKrishnaSreepathi are we ready to include the new variable naming format? If not, could we reach an agreement asap on this?

### CODE REVIEW

File: docs/developer-walk-through.rst [hunk 1]
```diff
@@ -1,140 +1,14 @@
...
-======================
+Other stuff
+===========
 
 .. toctree::
    :maxdepth: 2
    :caption: Contents
 
-   main-developer-workflow
-   how-to-add-a-new-script-to-the-cea
```
Comment (daren-thomas) [inline docs/developer-walk-through.rst]: I think this chapter is pretty important for new developers (or as a reference for older developers) - where is it added again?
(speakig of `how-to-add-a-new-script-to-the-cea`)

maybe we can add these in a sort of knowledge-base style catalog?

File: docs/developer-walk-through.rst [hunk 2]
```diff
@@ -1,140 +1,14 @@
...
    the-configuration-file
    configuration-file-details
-   how-to-create-a-new-release
```
Comment (daren-thomas) [inline docs/developer-walk-through.rst]: where is this going? we kind of still need this information, but totally ok with moving it around.

File: docs/environment.yml [hunk 3]
```diff
@@ -1,118 +1,208 @@
-name: cityenergyanalyst_docs
```
Comment (daren-thomas) [inline docs/environment.yml]: I don't understand why you're changing this. Have you tested it to work on the readthedocs? I'd prefer this to be a separate issue (updating the environment.yml for read-the-docs) - the RTD build system is not a windows system and therefore the environment won't have the exact same versions (your changes add vc9_1 versions of some modules which don't exist in ubuntu as the compiler doesn't exist).

File: docs/how-to-add-a-new-script-to-the-cea.rst [hunk 4]
```diff
@@ -1,16 +1,15 @@
-Contributing
-=============
+How to add a new script
```
Comment (daren-thomas) [inline docs/how-to-add-a-new-script-to-the-cea.rst]: ah. i think this explains my previous question :-)

File: docs/how-to-add-a-new-script-to-the-cea.rst [hunk 5]
```diff
@@ -1,16 +1,15 @@
...
 So you want to extend the CEA? This guide will get you up and running!
 
 The main steps you need to take are:
 
-#. copy the template script and rename it
-#. update the module-level documentation and credits
-#. develop your script
+#. start with a script template.
+#. develop your script.
 #. add your script to the ``cli.config`` file
```
Comment (daren-thomas) [inline docs/how-to-add-a-new-script-to-the-cea.rst]: to be changed when the scripts.yml file changes... i can manually update no problem.

File: docs/how-to-contribute.rst [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+
+The main steps you need to take are:
+
+#. Create an User Story or issue in Github.
+#. Create branch and learn how to use Github.
+#. get acquainted with variable naming.
+#. create your contribution.
+#. test your creation.
+#. Create a pull request on Github.
+#. Claim your CEA T-shirt!
```
Comment (daren-thomas) [inline docs/how-to-contribute.rst]: yay! t-shirts for everyone!

File: docs/developer-walk-through.rst (hunk 1 above)
Comment (JIMENOFONSECA) [reply docs/developer-walk-through.rst]: yes it is kept already but re-organized

File: docs/developer-walk-through.rst (hunk 2 above)
Comment (JIMENOFONSECA) [reply docs/developer-walk-through.rst]: yes it is kept already but re-organized

File: docs/how-to-add-a-new-script-to-the-cea.rst (hunk 4 above)
Comment (JIMENOFONSECA) [reply docs/how-to-add-a-new-script-to-the-cea.rst]: :-)

File: docs/environment.yml (hunk 3 above)
Comment (JIMENOFONSECA) [reply docs/environment.yml]: oh got you. I need your help with making this run

File: docs/how-to-add-a-new-script-to-the-cea.rst (hunk 5 above)
Comment (JIMENOFONSECA) [reply docs/how-to-add-a-new-script-to-the-cea.rst]: please do so, thanks!

File: docs/how-to-contribute.rst (hunk 6 above)
Comment (JIMENOFONSECA) [reply docs/how-to-contribute.rst]: :-D

**Comment by JIMENOFONSECA** [comment]
@daren-thomas so the PR is here since 24 days, we kind of need it to pass for Bhargava to include the new standard of variables. Would you mind checking the issue with the environment and get this baby up and running?

**Comment by daren-thomas** [comment]
@JIMENOFONSECA sure. On the same page. I was just waiting for your comments ;-) (I'll have time later this week)

**Pull request merged**